plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -302,6 +302,20 @@ private function update_pagination_anchors( $processor, $class_name, $key_prefix
 			if ( in_array( $class_name, array( 'wp-block-query-pagination-next', 'wp-block-query-pagination-previous' ), true ) ) {
 				$processor->set_attribute( 'data-wc-watch', 'callbacks.prefetch' );
 				$processor->set_attribute( 'data-wc-on--mouseenter', 'actions.prefetchOnHover' );
+			} else {
+				$inner_text = __( 'Page ', 'woocommerce' );
+				$processor->set_bookmark( 'page_number' );
+				while ( $processor->next_token() ) {
+					if ( '#text' === $processor->get_token_type() ) {
+						$page = trim( $processor->get_modifiable_text() );
+						if ( is_numeric( $page ) ) {
+							$inner_text .= $page;
```

Comment:
While I'm assuming this is unlikely, if there's ever a chance the `$page` variable isn't a numeric value, we end up setting the `aria-label` to `Page ` (with an extra space on the end). Thinking we may want to add that space when we combine the text with the page number to avoid that, something like:
```suggestion
				$inner_text = __( 'Page', 'woocommerce' );
				$processor->set_bookmark( 'page_number' );
				while ( $processor->next_token() ) {
					if ( '#text' === $processor->get_token_type() ) {
						$page = trim( $processor->get_modifiable_text() );
						if ( is_numeric( $page ) ) {
							$inner_text .= ' ' . $page;
```